<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterfaceCreator</class>
 <widget class="QDialog" name="UserInterfaceCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_add_child">
       <property name="text">
        <string>Add to Directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <widget class="QLineEdit" name="lineEdit_new_directory"/>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>Main Directory</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QPushButton" name="pushButton_removedirectory">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_add_display">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pushButton_remove_display">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_down">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_up">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QPushButton" name="pushButton_adddirectory">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" colspan="3">
      <widget class="QListWidget" name="listWidget_display"/>
     </item>
     <item row="0" column="4">
      <widget class="QListWidget" name="listWidget_nodes_directories"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserInterfaceCreator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserInterfaceCreator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
